.github/workflows/documentation_workflow.yml

```
@@ -7,7 +7,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Setup R
-        uses: r-lib/actions/setup-r@old
+        uses: r-lib/actions/setup-r@v2
```

Should probably be `v2-branch` (see [this link](https://github.com/r-lib/actions/tree/v2-branch/setup-r)).
```suggestion
        uses: r-lib/actions/setup-r@v2-branch
```